slack-api-model/src/test/java/test_locally/api/model/block/BlockKitTest.java

```diff
@@ -497,6 +497,126 @@ public void parseRichTextOnes3() {
         assertThat(block.getElements().size(), is(7));
     }
 
+    @Test
+    public void parseRichTextOnes4() {
```

Comment:
Before this PR's changes, this test case used to fail with the strict validation for testing.